src/models/components.d.ts

```diff
@@ -192,6 +192,7 @@ interface AutoCompleteSearchProps extends onSelectCategoryType {
   categoryOptions: Array<SearchCategoryItemType>;
   recommendations?: Array<recommendationType>;
   setIsMobileSearchOpen: Dispatch<SetStateAction<boolean>>;
+  selectedMobileItem: string;
```

Comment:
What is the purpose of this property?

Reply:
It gives us the selected tab item which can be either Food, Library etc <SearchCategories
                  onSelectCategory={props.onSelectCategory}
                  categoryOptions={props.categoryOptions}
                />

Reply:
Updated to to this selectedSearchCategory